package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "blog-tfa-node",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "node ./bin/www",
+    "test": "export NODE_ENV=test && export TWILIO_ACCOUNT_SID=my-account-sid && export TWILIO_AUTH_TOKEN=my-auth-token && export TWILIO_NUMBER=my-twilio-number && export APP_SECRET=your-secret && mocha spec --recursive",
```

Comment:
Nitpick: To do not pollute your environment with env variables, you can change this line by:

```javascript
"test": "NODE_ENV=test TWILIO_ACCOUNT_SID=my-account-sid TWILIO_AUTH_TOKEN=my-auth-token TWILIO_NUMBER=my-twilio-number APP_SECRET=your-secret mocha spec --recursive"
```